Firestore/core/src/remote/grpc_connection.cc

```diff
@@ -173,16 +198,7 @@ std::unique_ptr<grpc::ClientContext> GrpcConnection::CreateContext(
     context->AddMetadata(kAuthorizationHeader, absl::StrCat("Bearer ", token));
   }
 
-  // TODO(dimond): This should ideally also include the gRPC version, however,
```

Comment:
The concern that this TODO raises doesn't seem true to me -- can you think of a case where a user would be able to link a different version of gRPC at runtime?

Reply:
This concern dates back from when we were building Firestore in google3. The issue was that we would build our binary SDK against the gRPC in google3 and ship with undefined symbols for gRPC that would actually be supplied via CocoaPods on the end developer machine.

Now that we're distributed the way are, this concern no longer holds, so we could use the gRPC version.